internal/http/request.go

```diff
@@ -18,18 +18,18 @@
 package http
 
 import (
+	"context"
 	"encoding/binary"
+	"github.com/api7/ext-plugin-proto/go/A6"
```

Comment:
Let's configure your environment to make sure 3 party imports are sorted separated.